crates/saya/core/src/prover/stone_image.rs

```diff
@@ -51,8 +51,8 @@ impl StoneProver {
         static STONE_PROVER: OnceCell<(anyhow::Result<String>, anyhow::Result<String>)> =
             OnceCell::const_new();
 
-        let source = "piniom/state-diff-commitment";
-        let verifier = "piniom/verifier:latest";
+        let source = "neotheprogramist/state-diff-commitment:latest";
+        let verifier = "neotheprogramist/verifier:latest";
```

Comment:
We will configure GitHub container registry for `dojo-os` to have those being automatically published if we can from the CI.

Reply:
do we want to wait with merging until it's configured or fix in another PR?